pkg/features/features.go

```diff
@@ -24,6 +24,9 @@ const (
 	//
 	// Allow user set one time batch stratey in rollout annotation
 	OneTimeStrategy featuregate.Feature = "OneTimeStrategy"
+
+	// Allow filter rollout class when reconcile
+	RolloutClassFilter featuregate.Feature = "RolloutClassFilter"
```

Comment:
1. change this feature name to RolloutClassPredicate
2. add the feature owner in comment